packages/forma-36-react-components/src/components/Card/EntryCard/EntryCard.css

```diff
@@ -18,8 +18,19 @@
       var(--transition-easing-default);
 }
 
+.EntryCard__header {
+  height: calc(1rem * (45 / var(--font-base-default)));
```

Comment:
can we do `44px` instead of `45px` to align with the 4px grid?

Reply:
I think we can

Reply:
updated